app/controllers/admin/court_applications_controller.rb

```diff
@@ -1,8 +1,32 @@
 module Admin
   class CourtApplicationsController < Admin::ApplicationController
+    def index
+      @court_applications = if params[:query].present?
+                              CourtApplication.search(params[:query]).order(created_at: :desc).page(params[:page])
+                            else
+                              CourtApplication.order(created_at: :desc).page(params[:page])
+                            end
+    end
+
+    def new
+      @court_application = FactoryBot.build(:court_application)
```

Comment:
wow, I've never used `FactoryBot` in a not testing code. Just surprised nothing against it.

Reply:
We are already using this in the application